www/src/pages/ar/faq.md

```diff
@@ -6,9 +6,10 @@ lang: ar
 dir: rtl
 ---
 
-ها هي بعض الأسئلة التي يَكثر سُؤالها عن `create-t3-app`.
 
-## ماذا بعد؟ كيف استعمل `create-t3-app` لأنشاء تطبيق؟
+هاك ما يَكثُر من سٌؤاله عن `create-t3-app`:
+
+## كيف استعمل `create-t3-app` ؟
```

Comment:
Although using `هاك` is correct as it means look and I do like it more than my suggestion, it can easily be misunderstood as something bad and searching it out will lead to searching for hacks about valorant. https://duckduckgo.com/?q=%D9%87%D8%A7%D9%83&ia=web.

What about using `إليك`?

---

Here is the section in English, because I may be wrong in understanding it.

https://create.t3.gg/en/faq#whats-next-how-do-i-make-an-app-with-this

---

I think that section is talking about what to do, after using `npm create t3-app@latest`. So removing the context `ماذا بعد` changes the context.

```suggestion
إليك ما يَكثُر من سٌؤاله عن `create-t3-app`.

## ماذا بعد؟ كيف استعمل `create-t3-app` لأنشاء تطبيق؟
```

Reply:
👍🏼